tests/tests_common_test.go

```diff
@@ -412,3 +414,23 @@ func addDomainResourcesToVMI(vmi *kubevirtv1.VirtualMachineInstance, cpuCores ui
 	}
 	return vmi
 }
+
+func expectRunbookURLs() {
+	promRuleObj := metrics.NewPrometheusRule(strategy.GetNamespace())
```

Comment:
This is a functional test, so I think it would be better to check the runbook URLs on actual PrometheusRule resource that was created by the operator.

Can you change this test, so that it fetches the PrometheusRule from the cluster?

Reply:
Once you inline this function into `metrics_test.go`, you can get the rule from cluster like this:
```go
promRule := &promv1.PrometheusRule{}
Expect(apiClient.Get(ctx, prometheusRuleRes.GetKey(), promRule)).To(Succeed())
```

Similarly how it is done in test case: `[test_id:4665] should create prometheus rule`.